### ISSUE TITLE
error when composer update

### ISSUE THREAD

**Issue by hopewise** [rappasoft/laravel-5-boilerplate#295]
Although this is mainly an issue at my side, but I thought you can help me with it, as I am using this lovely repo :)

```
C:\Bitnami\wappstack-5.6.18-0\apps\project>composer update
> php artisan clear-compiled
Loading composer repositories with package information
Updating dependencies (including require-dev)
Your requirements could not be resolved to an installable set of packages.

  Problem 1
    - Conclusion: don't install laravel/framework v5.2.16
    - Conclusion: don't install laravel/framework v5.2.15
    - Conclusion: remove laravel/framework v5.2.14
    - Conclusion: don't install laravel/framework v5.2.14
    - Conclusion: don't install laravel/framework v5.2.13
    - Conclusion: don't install laravel/framework v5.2.12
    - Conclusion: don't install laravel/framework v5.2.11
    - Conclusion: don't install laravel/framework v5.2.10
    - Conclusion: don't install laravel/framework v5.2.9
    - Conclusion: don't install laravel/framework v5.2.8
    - Conclusion: don't install laravel/framework v5.2.7
    - Conclusion: don't install laravel/framework v5.2.6
    - Conclusion: don't install laravel/framework v5.2.5
    - Conclusion: don't install laravel/framework v5.2.4
    - Conclusion: don't install laravel/framework v5.2.3
    - Conclusion: don't install laravel/framework v5.2.2
    - Conclusion: don't install laravel/framework v5.2.1
    - laravelcollective/html v5.1.0 requires illuminate/session 5.1.* -> satisfiable by illuminate/session[v5.1.1, v5.1.13, v5.1.16, v5.1.2, v5.1.20, v5.1.22, v5.1.25, v5.1.28, v5.1.6, v5.1.8].
    - laravelcollective/html v5.1.1 requires illuminate/session 5.1.* -> satisfiable by illuminate/session[v5.1.1, v5.1.13, v5.1.16, v5.1.2, v5.1.20, v5.1.22, v5.1.25, v5.1.28, v5.1.6, v5.1.8].
    - laravelcollective/html v5.1.2 requires illuminate/session 5.1.* -> satisfiable by illuminate/session[v5.1.1, v5.1.13, v5.1.16, v5.1.2, v5.1.20, v5.1.22, v5.1.25, v5.1.28, v5.1.6, v5.1.8].
    - laravelcollective/html v5.1.3 requires illuminate/session 5.1.* -> satisfiable by illuminate/session[v5.1.1, v5.1.13, v5.1.16, v5.1.2, v5.1.20, v5.1.22, v5.1.25, v5.1.28, v5.1.6, v5.1.8].
    - laravelcollective/html v5.1.4 requires illuminate/session 5.1.* -> satisfiable by illuminate/session[v5.1.1, v5.1.13, v5.1.16, v5.1.2, v5.1.20, v5.1.22, v5.1.25, v5.1.28, v5.1.6, v5.1.8].
    - laravelcollective/html v5.1.5 requires illuminate/session 5.1.* -> satisfiable by illuminate/session[v5.1.1, v5.1.13, v5.1.16, v5.1.2, v5.1.20, v5.1.22, v5.1.25, v5.1.28, v5.1.6, v5.1.8].
    - laravelcollective/html v5.1.6 requires illuminate/session 5.1.* -> satisfiable by illuminate/session[v5.1.1, v5.1.13, v5.1.16, v5.1.2, v5.1.20, v5.1.22, v5.1.25, v5.1.28, v5.1.6, v5.1.8].
    - laravelcollective/html v5.1.7 requires illuminate/session 5.1.* -> satisfiable by illuminate/session[v5.1.1, v5.1.13, v5.1.16, v5.1.2, v5.1.20, v5.1.22, v5.1.25, v5.1.28, v5.1.6, v5.1.8].
    - laravelcollective/html v5.1.8 requires illuminate/session 5.1.* -> satisfiable by illuminate/session[v5.1.1, v5.1.13, v5.1.16, v5.1.2, v5.1.20, v5.1.22, v5.1.25, v5.1.28, v5.1.6, v5.1.8].
    - laravelcollective/html v5.1.9 requires illuminate/session 5.1.* -> satisfiable by illuminate/session[v5.1.1, v5.1.13, v5.1.16, v5.1.2, v5.1.20, v5.1.22, v5.1.25, v5.1.28, v5.1.6, v5.1.8].
    - don't install illuminate/session v5.1.1|don't install laravel/framework v5.2.0
    - don't install illuminate/session v5.1.13|don't install laravel/framework v5.2.0
    - don't install illuminate/session v5.1.16|don't install laravel/framework v5.2.0
    - don't install illuminate/session v5.1.2|don't install laravel/framework v5.2.0
    - don't install illuminate/session v5.1.20|don't install laravel/framework v5.2.0
    - don't install illuminate/session v5.1.22|don't install laravel/framework v5.2.0
    - don't install illuminate/session v5.1.25|don't install laravel/framework v5.2.0
    - don't install illuminate/session v5.1.28|don't install laravel/framework v5.2.0
    - don't install illuminate/session v5.1.6|don't install laravel/framework v5.2.0
    - don't install illuminate/session v5.1.8|don't install laravel/framework v5.2.0
    - Installation request for laravel/framework 5.2.* -> satisfiable by laravel/framework[v5.2.0, v5.2.1, v5.2.10, v5.2.11, v5.2.12, v5.2.13, v5.2.14, v5.2.15, v5.2.16, v5.2.2, v5.2.3, v5.2.4, v5.2.5, v5.2.6, v5.2.7, v5.2.8, v5.2.9].
    - Installation request for laravelcollective/html 5.1.* -> satisfiable by laravelcollective/html[v5.1.0, v5.1.1, v5.1.2, v5.1.3, v5.1.4, v5.1.5, v5.1.6, v5.1.7, v5.1.8, v5.1.9].

```

so, how to solve this?

**Comment by error500** [comment]
It says
    Your requirements could not be resolved to an installable set of packages.
It might be php/mysql requirements ?
https://laravel.com/docs/5.2#server-requirements
What are your php version ? do you have the extensions installed ?

**Comment by hopewise** [comment]
I am using this mage: https://github.com/mtmacdonald/docker-laravel

Could it be related to the correct permissions to directories and files?

I will check the requirements if they all met in the image..

**Comment by hopewise** [comment]
# php -v
PHP 5.5.9-1ubuntu4.14 (cli) (built: Oct 28 2015 01:34:46)
Copyright (c) 1997-2014 The PHP Group
Zend Engine v2.5.0, Copyright (c) 1998-2014 Zend Technologies
    with Zend OPcache v7.0.3, Copyright (c) 1999-2014, by Zend Technologies
    with Xdebug v2.2.3, Copyright (c) 2002-2013, by Derick Rethans


I have cloned the repo, I am using a dedicated environment (https://github.com/mtmacdonald/docker-laravel) 

composer install will result into:

```
Problem 1
    - Conclusion: don't install laravel/framework v5.2.19
    - Conclusion: don't install symfony/finder v3.0.2
    - Conclusion: don't install laravel/framework v5.2.18
    - Conclusion: don't install symfony/finder v3.0.1
    - Conclusion: don't install laravel/framework v5.2.17
    - Conclusion: don't install symfony/finder v3.0.0
    - Conclusion: don't install laravel/framework v5.2.16
    - Conclusion: don't install symfony/finder v2.8.2
    - Conclusion: don't install laravel/framework v5.2.15
    - Conclusion: don't install symfony/finder v2.8.1
    - Installation request for laravelcollective/html 5.1.* -> satisfiable by laravelcollective/html[v5.1.0, v5.1.1, v5.1.2, v5.1.3, v5.1.4, v5.1.5, v5.1.6, v5.1.7, v5.1.8, v5.1.9].
    - Conclusion: don't install laravel/framework v5.2.14
    - laravel/framework v5.2.0 requires symfony/finder 2.8.*|3.0.* -> satisfiable by symfony/finder[v2.8.0, v2.8.1, v2.8.2, v3.0.0, v3.0.1, v3.0.2].
    - laravel/framework v5.2.1 requires symfony/finder 2.8.*|3.0.* -> satisfiable by symfony/finder[v2.8.0, v2.8.1, v2.8.2, v3.0.0, v3.0.1, v3.0.2].
    - laravel/framework v5.2.10 requires symfony/finder 2.8.*|3.0.* -> satisfiable by symfony/finder[v2.8.0, v2.8.1, v2.8.2, v3.0.0, v3.0.1, v3.0.2].
    - laravel/framework v5.2.11 requires symfony/finder 2.8.*|3.0.* -> satisfiable by symfony/finder[v2.8.0, v2.8.1, v2.8.2, v3.0.0, v3.0.1, v3.0.2].
    - laravel/framework v5.2.12 requires symfony/finder 2.8.*|3.0.* -> satisfiable by symfony/finder[v2.8.0, v2.8.1, v2.8.2, v3.0.0, v3.0.1, v3.0.2].
    - laravel/framework v5.2.13 requires symfony/finder 2.8.*|3.0.* -> satisfiable by symfony/finder[v2.8.0, v2.8.1, v2.8.2, v3.0.0, v3.0.1, v3.0.2].
    - laravel/framework v5.2.2 requires symfony/finder 2.8.*|3.0.* -> satisfiable by symfony/finder[v2.8.0, v2.8.1, v2.8.2, v3.0.0, v3.0.1, v3.0.2].
    - laravel/framework v5.2.3 requires symfony/finder 2.8.*|3.0.* -> satisfiable by symfony/finder[v2.8.0, v2.8.1, v2.8.2, v3.0.0, v3.0.1, v3.0.2].
    - laravel/framework v5.2.4 requires symfony/finder 2.8.*|3.0.* -> satisfiable by symfony/finder[v2.8.0, v2.8.1, v2.8.2, v3.0.0, v3.0.1, v3.0.2].
    - laravel/framework v5.2.5 requires symfony/finder 2.8.*|3.0.* -> satisfiable by symfony/finder[v2.8.0, v2.8.1, v2.8.2, v3.0.0, v3.0.1, v3.0.2].
    - laravel/framework v5.2.6 requires symfony/finder 2.8.*|3.0.* -> satisfiable by symfony/finder[v2.8.0, v2.8.1, v2.8.2, v3.0.0, v3.0.1, v3.0.2].
    - laravel/framework v5.2.7 requires symfony/finder 2.8.*|3.0.* -> satisfiable by symfony/finder[v2.8.0, v2.8.1, v2.8.2, v3.0.0, v3.0.1, v3.0.2].
    - laravel/framework v5.2.8 requires symfony/finder 2.8.*|3.0.* -> satisfiable by symfony/finder[v2.8.0, v2.8.1, v2.8.2, v3.0.0, v3.0.1, v3.0.2].
    - laravel/framework v5.2.9 requires symfony/finder 2.8.*|3.0.* -> satisfiable by symfony/finder[v2.8.0, v2.8.1, v2.8.2, v3.0.0, v3.0.1, v3.0.2].
    - Conclusion: don't install symfony/finder v2.8.0
    - Installation request for laravel/framework 5.2.* -> satisfiable by laravel/framework[v5.2.0, v5.2.1, v5.2.10, v5.2.11, v5.2.12, v5.2.13, v5.2.14, v5.2.15, v5.2.16, v5.2.17, v5.2.18, v5.2.19, v5.2.2, v5.2.3, v5.2.4, v5.2.5, v5.2.6, v5.2.7, v5.2.8, v5.2.9].
```

**Comment by blomdahldaniel** [comment]
This is a weird error. Have you gotten any success? Do you get the same error when trying to install just the plain Laravel application? If that is the case I suggest that you open a question in the Laravel forums. I will close this issue since it has most likely nothing to do with this repo in particular.

**Comment by hopewise** [comment]
yes, its related to my code, I will start with a fresh install of repo and
check.. thanks

On Mon, Feb 22, 2016 at 3:54 PM, Daniel Blomdahl <notifications@github.com>
wrote:

> Closed #295
> <https://github.com/rappasoft/laravel-5-boilerplate/issues/295>.
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/rappasoft/laravel-5-boilerplate/issues/295#event-559644492>
> .
>



-- 
--
Kind Regards,
---------------------------------------------
Samir Sabri
Software Architect& Developer
www.dcaclab.com
Jordan-Middle East
